### DISCUSSION TITLE
Project Status?

### DISCUSSION THREAD

**Discussion by brongulus** [vedang/pdf-tools#306] [General / FAQ]
Hi @vedang 

First of all, thanks a lot for taking the huge undertaking of being the maintainer of this awesome project. Appreciate it a lot!

However I've noticed that it's been a long time since a lot of the PRs have been merged/there's been any commit activity. Just wanted to know if it's life keeping you busy or something else? 

Would really love for PRs like #230 #224 etc to be merged!

Cheers

**Comment by vedang** [comment]
Hi @brongulus ! Life's keeping me really busy unfortunately. I can't make much time for pdf-tools in the near future. Hopefully I'll get a break towards the end of May, which is when I'll take next steps on this project, one way or the other.
